docs/backups.md

```diff
@@ -449,6 +450,14 @@ $ kubectl apply -f deploy/restore.yaml
 Point-in-time recovery functionality allows users to revert the database back to
 a state before an unwanted change had occurred.
 
+!!! note
+
+    For this feature to work, the Operator initiates a full backup 
+    immediately after the cluster creation, to use it as a basis for
+    point-in-time recovery when needed. Because this backup has a technical
```

Comment:
let's not say it's not listed because it has technical purpose.

the reality is this backup is started by crunchy controller and they're not using our custom resources for backups. that's the reason you can't see it in the list.

i think just mentioning this backup won't be visible in the list is enough